.github/workflows/tests.yml

```diff
@@ -402,21 +366,87 @@ jobs:
           ls -lhR benchmarks
           bash ci/scripts/combine-dbs.sh
 
-      - name: Upload benchmark db
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}
-          AWS_DEFAULT_REGION: us-east-2  # this is needed for boto for some reason
-          DB_NAME: benchmark.db
-        run: |
-          aws s3 cp $DB_NAME s3://coiled-runtime-ci/benchmarks/
+      # - name: Upload benchmark db
+      #   env:
+      #     AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
+      #     AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}
+      #     AWS_DEFAULT_REGION: us-east-2  # this is needed for boto for some reason
+      #     DB_NAME: benchmark.db
+      #   run: |
+      #     aws s3 cp $DB_NAME s3://coiled-runtime-ci/benchmarks/
```

Comment:
Thank you for being careful here :)

Reply:
@ian-r-rose is it safe to uncomment this now? Unless there are any other things you want to check?

Reply:
I think we're ready